dev/com.ibm.ws.security.oidc.client_fat.jaxrs/fat/src/com/ibm/ws/security/openidconnect/client/fat/jaxrs/CommonTests/JaxRSClientAPITests.java

```diff
@@ -111,11 +108,61 @@ protected TestSettings updateMap(TestSettings settings, String theKey, String th
         }
         Log.info(thisClass, "updateMap", "Processing Key: " + theKey + " Value: " + theValue);
         currentMap.put(theKey, theValue);
+        if (Constants.JWE_TOKEN_FORMAT.equals(FATSuite.repeatFlag)) {
+            currentMap.put("skipApiChecks", "true");
+        }
         settings.setRequestParms(currentMap);
 
         return settings;
     }
 
+    /**
+     * Create and Save a JWS token that the test userinfo endpoint can return - this will
+     * allow us to test that the RP and RS can process a userinfo reponse that contains
+     * a JWS instead of a simple JSON reponse
+     */
+    public void saveJWSUserInfoResponse() throws Exception {
+
+        String thisMethod = "saveJWSUserInfoResponse";
+        msgUtils.printMethodName(thisMethod);
+
+        JWTTokenBuilder builder = new JWTTokenBuilder();
+
+        builder.setIssuer(testOPServer.getHttpString() + "/TokenEndpointServlet");
+        builder.setAlorithmHeaderValue(Constants.SIGALG_RS256);
+        builder.setRSAKey(testOPServer.getServer().getServerRoot() + "/RS256private-key.pem");
+        builder.setIssuer("testuser");
```

Comment:
Setting the issuer twice?

Reply:
I can clean this up when I add the testing for userinfo (it's a method used just for the userinfo endpoint)